nemo/collections/asr/data/audio_to_text.py

```diff
@@ -653,6 +653,7 @@ def __init__(
         use_start_end_token: bool = True,
         return_sample_id: bool = False,
         channel_selector: Optional[ChannelSelectorType] = None,
+        language: Optional[str] = None,
```

Comment:
Could you please also add this parameter to the docstring of the class?

Reply:
I have updated it